website/blog/2025-12-05-acr-abac-repository-permissions/index.md

```diff
@@ -0,0 +1,77 @@
+
+ACR registries support a permissions mode called "**RBAC Registry + ABAC Repository Permissions**" that makes them ABAC-enabled. Once configured, registry administrators add ABAC conditions to standard Azure RBAC role assignments, scoping permissions to specific repositories or namespace prefixes. This enables:
+
+* **CI/CD pipelines** to push images only to their approved namespaces and repositories
+* **AKS clusters, Azure Container Apps, and Azure Container Instances** to pull only from authorized repositories
+* **Microsoft Entra identities** to enforce permission boundaries through standard role assignments
+
+## Enabling ABAC on ACR
+
+ABAC can be enabled on all new and existing ACR registries across all SKUs, either during registry creation or configured on existing registries. You can assign RBAC roles with optional ABAC conditions to users, groups, service principals, and managed identities—including AKS kubelet and workload identities, Azure Container Apps identities, and Azure Container Instances identities.
+
+Here is the Azure Portal experience for enabling ABAC on a new ACR during creation:
+
+![Enabling ABAC on a new ACR during Creation](./acr-enabling-abac-during-acr-create.png)
+
+Here is the Azure Portal experience for enabling ABAC on an existing ACR:
+
+![Enabling ABAC on an existing ACR](./acr-enabling-abac-during-acr-update.png)
+
+ABAC can also be enabled on ACR registries through Azure Resource Manager (ARM), Bicep files, Terraform templates, and Azure CLI.
+
+## ABAC-enabled built-in roles
+
+Once a registry is ABAC-enabled (configured to "**RBAC Registry + ABAC Repository Permissions**"), registry admins can use these ABAC-enabled built-in roles to grant repository-scoped permissions:
+
+* **Container Registry Repository Reader**: grants image pull and metadata read permissions, including permissions for `HEAD` requests, `GET` manifest requests, `GET` layer blob requests, tag resolution, and discovering OCI referrers.
+* **Container Registry Repository Writer**: grants Repository Reader permissions, as well as image and tag push permissions.
+* **Container Registry Repository Contributor**: grants Repository Reader and Repository Writer permissions, as well as image and tag delete permissions.
+
+Note that these roles do not grant repository list permissions.
+
+* The separate **Container Registry Repository Catalog Lister** must be assigned to grant repository list permissions.
+* The **Container Registry Repository Catalog Lister** role does not support ABAC conditions in role assignments; assigning this role grants permissions to list all repositories in a registry.
+
+## Important role behavior changes in ABAC mode
+
+:::caution
+When a registry is ABAC-enabled by configuring its permissions mode to "**RBAC Registry + ABAC Repository Permissions**", existing built-in roles and role assignments will have different behaviors and will no longer grant the same set of permissions to ACR registries.
```

Comment:
can we mention what is the default authorization mode?

Reply:
Addressed